### PR TITLE
WIP: Cmi dummy file

### PR DESCRIPTION
**Pull request by bericyb**
@sbillinge 
Testing the Billinge group git workflow.

### CODE REVIEW

File: src/diffpy/pdfgui/gui/dummyfile.py [hunk 1]
```diff
@@ -0,0 +1 @@
+#Hi Simon Billinge. This is Beric's dummy branch.
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/dummyfile.py:1]: to play with the workflow, can you make edits here?

**Review by sbillinge — COMMENTED** [review]
Thanks @bericyb this looks good, well done!

Kudos for doing the PR into the `cmi_engine branch`.  That reminds me that we need to make a `cmi_mpdf` branch from that `cmi_engine` branch.

I made an inline comment

File: src/diffpy/pdfgui/gui/dummyfile.py (hunk 1 above)
Comment (bericyb) [reply src/diffpy/pdfgui/gui/dummyfile.py]: Done.

File: src/diffpy/pdfgui/gui/dummyfile.py [hunk 2]
```diff
@@ -1 +1,2 @@
-#Hi Simon Billinge. This is Beric's dummy branch. 
+#Hi Simon Billinge. This is Beric's dummy branch.
+# Making some secondary edits here.
```
Comment (bericyb) [inline src/diffpy/pdfgui/gui/dummyfile.py:2]: Made some new edits here.

Comment (sbillinge) [reply src/diffpy/pdfgui/gui/dummyfile.py:2]: nice.  Please don't forget to log this by responding to the previous comment though....

File: src/diffpy/pdfgui/gui/dummyfile.py (hunk 1 above)
Comment (bericyb) [reply src/diffpy/pdfgui/gui/dummyfile.py]: @sbillinge Sorry, forgot to mention you in my reply here. If everything looks good I'll close the PR. 
Thank you!

Comment (sbillinge) [reply src/diffpy/pdfgui/gui/dummyfile.py]: Yes, great job, I think you got it!  Looking forward to working with you!

**Pull request closed without merging**